src/components/proposals/proposal-view.vue

```diff
@@ -19,6 +19,7 @@ export default {
     IpfsImageViewer: () => import('~/components/ipfs/ipfs-image-viewer.vue'),
     IpfsFileViewer: () => import('~/components/ipfs/ipfs-file-viewer.vue'),
     ProposalDynamicPopup: () => import('~/components/proposals/proposal-dynamic-popup.vue')
+    // VersionHistory: () => import('~/components/proposals/version-history.vue') // temporarily hide
```

Comment:
@evgenibir Things for temporarily hide / show  would be good to mark with TODO, so that they can be seen in todo list in IDE

Reply:
fixed